src/Stack/Types/NamedComponent.hs

```diff
@@ -119,3 +132,4 @@ splitComponents =
   go s e t b (CExe x:xs) = go s (e . (x:)) t b xs
   go s e t b (CTest x:xs) = go s e (t . (x:)) b xs
   go s e t b (CBench x:xs) = go s e t (b . (x:)) xs
+  go s e t b x = go s e t b x
```

Comment:
@theobat, as the righthand side is the same as the lefthand side, how will this ever terminate? Presumably `x` would be `(CFlib x : xs)`, as the other patterns are catered for. I wondered if this was intended to be `go s e t b (CFlib x : xs) = go s e t b xs`?

Reply:
that's right, good catch, I'll fix it